pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -370,13 +376,30 @@ func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcont
 		resources.Memory = *resourceConfig.Memory
 	}
 	if resourceConfig.CpuShares != nil {
-		resources.CpuShares = *resourceConfig.CpuShares
-	}
-	if resourceConfig.CpuQuota != nil {
-		resources.CpuQuota = *resourceConfig.CpuQuota
+		if m.cgroupv2 {
```

Comment:
nit: Can we be consistent about nesting wrt checking m.cgroupv2 and checking if inputs are non-nil?  Here we do `if CpuShares != nil`, then `if m.cgroupv2`, and below we do the reverse.  Either one is fine, but consistency would be nice.

Reply:
sure, I've fixed it